keyboards/royal_kludge/rk68_rgb/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+OPT_DEFS += -DDYNAMIC_KEYMAP_LAYER_COUNT=3
```

Comment:
why do you need to define 3 layers for VIA? And if you do, there are cleaner ways to do it than through OPT_DEFS

Reply:
I googled because I was getting an error and found that configuration setting. This is the error if I remove that line:

`compiling: quantum/dynamic_keymap.c                                                                quantum/dynamic_keymap.c:92:1: error: static assertion failed: "Dynamic keymaps are configured to use more EEPROM than is available."`

Reply:
Should I simply add this to  config.h?

```c
#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

Reply:
> Should I simply add this to config.h?
> 
> ```c
> #define DYNAMIC_KEYMAP_LAYER_COUNT 3
> ```

you can either do that in the via `config.h`, or enable LTO in the via `rules.mk` 
```mk
LTO_ENABLE = yes
```
( will reduce the footprint, might just be enough)

Reply:
setting LTO_ENABLE to yes wasn't enough.